packages/web-runtime/src/pages/logout.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <div>
+    <p>Please wait while you're being logged out.</p>
+  </div>
+</template>
+<script>
+import { authService } from '../services/auth'
+export default {
+  name: 'LogoutPage',
+  created() {
+    authService.logoutUser()
```

Comment:
```suggestion
    await authService.logoutUser()
```

Reply:
The redirect to the login page should only ever happen after the logout has been finalized. Needs to be awaited.